S21/indraneelL/wordConvert.py

```diff
@@ -0,0 +1,29 @@
+import locale
+import re
+
+## read the file and iterate through every line
+class WordConvert:
+    def readTestFile(self):
+        textFile = open("tests.tsv", "r")
+        while textFile:
+            line = textFile.readline()
+            if line == "" or line == '\n':
+                break
+            word, lang, _  = line.split('\t')
+            self.methodToProcessText(word,lang)
+        textFile.close()
+
+    def makeWordsLower(self,word):
+        print(word.lower())
+
+    def methodToProcessText(self, text, lang):
+        # using regex check for the language and do functionalities
+        word = text
+        if re.findall("\A[n,t][AEIOUÁÉÍÓÚ]",text) and re.match("^ga.*",lang):
+            word = re.sub(r'(^[n,t])([AEIOUÁÉÍÓÚ])', r'\1-\2',text)
+        elif re.match("^tr.*", lang):
```

Comment:
Although it doesn't change the functionality of the code. Traditional Python writing suggests us to use elif between if and else.  And could you please add a comment explaining the this line of code?

Reply:
Yes, here we have used regex to find the first small letter and following with any of the vowel in the regex [] operator and making sure to taking the first letter and second letter into the subset and adding a hyphen in between them.